src/afl-forkserver.c

```diff
@@ -213,7 +214,7 @@ read_s32_timed(s32 fd, s32 *buf, u32 timeout_ms, volatile u8 *stop_soon_p) {
 static void afl_fauxsrv_execv(afl_forkserver_t *fsrv, char **argv) {
 
   unsigned char tmp[4] = {0, 0, 0, 0};
-  pid_t         child_pid = -1;
+  pid_t         child_pid;
```

Comment:
Why remove the safe initial assignment? It doesn't hurt to have it as a safeguard for future refactorings (and the compiler will throw it out)

Reply:
I guess heiko removed it because it cannot be uninitialized in the function. could have been for a removal for "dead assignment" warning.